context/app/static/js/components/Routes/useSendWebVitals.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react';
+import ReactGA from 'react-ga';
+import { getCLS, getFID, getLCP, getTTFB } from 'web-vitals';
```

Comment:
In the docs, `TTFB` isn't highlighted the way the other three are... Do you have any sense why not?

Reply:
If you look in the beginning of the readme they list those three as 'core'.